src/lispmach.h

```diff
@@ -436,6 +436,11 @@ list_ref (repv list, int elt)
 #define SP_REG asm("%l1")
 #define SLOTS_REG asm("%l2")
 #endif
+#ifdef __loongarch__
+#define PC_REG asm("$r23")
+#define SP_REG asm("$r24")
+#define SLOTS_REG asm("$r25")
```

Comment:
Bare register indexes are hard to remember and work with, it would be better to write using ABI names:

```c
#define PC_REG asm("$s0")
#define SP_REG asm("$s1")
#define SLOTS_REG asm("$s2")
```

Reply:
how can I do benchmark for this .